### ISSUE TITLE
Error creating a mandatory not null property

### ISSUE THREAD

**Issue by creisle** [orientechnologies/orientjs#355]
orientjs version: 3.0.2
orientdb version: 3.0.6

I am running into an error creating non-null properties. 

```text
TypeError: (intermediate value)(intermediate value)(intermediate value).all is not a function
```

I think this may be a typo in the property creation logic
/orientjs/lib/client/database/class/property.js

line 202 seems to have the all not on the db command like the other property statements
```js
if (property.notNull !== undefined) {
    promises.push(
      this.db.command(
        prefix + "NOTNULL " + (property.notNull ? "true" : "false").all()
      )
    );
  }
```

whereas I would have expected

```js
if (property.notNull !== undefined) {
    promises.push(
      this.db.command(
        prefix + "NOTNULL " + (property.notNull ? "true" : "false")
      ).all()
    );
  }
```

**Comment by wolf4ood** [comment]
Hi @creisle 

thanks for raising this issue. Indeed there was a typo. Fixed and pushed.
It will be available in the next OrientJS 3.0.3 release

**Comment by wolf4ood** [comment]
Hi @creisle 

we have just release OrientJS 3.0.3 with the fix.
Thanks
